giskard/rag/testset.py

```diff
@@ -154,12 +158,12 @@ def push_to_hub(
         template = template_path.read_text()
 
         # Make and push the dataset card
-        # global _default_llm_model
-        config = {
-            "metadata": {
-                "model": "gpt-4o"
-            }
-        }
+        llm_client = get_default_client()
+        if not hasattr(llm_client, "get_config") or not callable(getattr(llm_client, "get_config")):
```

Comment:
I don't think this is needed because we are adding it as abstractmethod to an ABC, right?